reana_client/api/client.py

```diff
@@ -131,8 +131,84 @@ def create_workflow(reana_specification, name, access_token):
         raise e
 
 
+def create_workflow_from_json(workflow_json, name, access_token,
+                              parameters=None, workflow_engine='yadage',
+                              outputs=None):
+    """Create a workflow from json specification.
+
+    :param workflow_json: workflow specification in json format.
+    :param name: name or UUID of the workflow to be started.
+    :param access_token: Access token of the current user.
+    :param parameters: workflow input parameters dictionary.
+    :param workflow_engine: one of the workflow engines (yadage, serial, cwl)
+    :param outputs: dictionary with expected workflow outputs.
+
+    :Example:
+
+         >>> create_workflow_from_json(
```

Comment:
👍 ! I am not getting this displayed when building the docs though.

Reply:
<img width="606" alt="screenshot 2019-01-24 at 13 37 32" src="https://user-images.githubusercontent.com/5921114/51678676-448f8280-1fdd-11e9-9561-7c4a14aef89a.png">
Does it look like this?

Reply:
I am doing: 
```console
$ ~/reana-client git:(8e1aa70) $ python setup.py build_sphinx
running build_sphinx
Running Sphinx v1.8.2
making output directory...
building [mo]: all of 0 po files
building [html]: all source files
updating environment: 11 added, 0 changed, 0 removed
reading sources... [100%] userguide
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] userguide
generating indices... genindex
writing additional pages... search
copying static files... done
copying extra files... done
dumping search index in English (code: en) ... done
dumping object inventory... done
build succeeded.

The HTML pages are in docs/_build/html.
```
![image](https://user-images.githubusercontent.com/9035606/51678900-c384bb00-1fdd-11e9-950f-73ca3efe5e0f.png)